### PR TITLE
Don't rely on dependent data being defined to build Financials

### PR DESCRIPTION
**Pull request by markolson**

### CODE REVIEW

File: src/server/enrollment-system.js [hunk 1]
```diff
@@ -904,10 +904,13 @@ function veteranToFinancialsInfo(veteran) {
...
   const dependentFinancials = veteranToDependentFinancialsCollection(veteran);
-  const hasDependentFinancials = _.compact(dependentFinancials.dependentFinancials.map((child) => { return child.income; })).length > 0;
+  let hasDependentFinancials = false;
+  if (dependentFinancials) {
+    hasDependentFinancials = _.compact(dependentFinancials.dependentFinancials.map((child) => { return child.incomes; })).length > 0;
+  }
   const spouseFinancials = veteranToSpouseFinancials(veteran);
 
-  const hasIncomeData = expenses || incomes || spouseFinancials.incomes || hasDependentFinancials;
+  const hasIncomeData = expenses || incomes || spouseFinancials.spouseFinancials.incomes || hasDependentFinancials;
```
Comment (akainic) [inline src/server/enrollment-system.js]: That's weird, I thought this change was already merged into master...

Comment (markolson) [inline src/server/enrollment-system.js]: Sorry, I'm having to rebase because I branched from a WIP branch that wasn't up to date.

**Comment by akainic** [comment]
LGTM! I'll merge once tests pass.

**Comment by robbiethegeek** [comment]
:shipit:  👍 :)

**Pull request merged**